tests/cypress/e2e/ITILObject/user_mentions.cy.js

```diff
@@ -1,143 +0,0 @@
-/**
```

Comment:
Any reason this test was removed ?

It was far from great but it still tested some user interactions that can't be done with a backend test so maybe we should keep it around for now ?

Reply:
Which user interactions were important to test like this? Is it just the loading of the user list after typing `@`?

Reply:
I think this test was made to ensure that user mention restiction was working as expected. A jest test cannot validate that.